package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "base-app",
-  "version": "1.11.0",
+  "version": "1.13.0",
```

Comment:
any reason we're going up 2 versions?

Reply:
Great question - I must have forgotten to update the version last time as it was out of sync with the Changelog. I updated in the PR